midealocal/cloud.py

```diff
@@ -408,7 +408,25 @@ async def list_appliances(
         return None
 
     async def get_device_info(self, device_id: int) -> dict[str, Any] | None:
-        """Get device information."""
+        r"""Get device information.
```

Comment:
```suggestion
        """Get device information.
```

Reply:
oh, thanks for your review comments, seems without `r`, it can't accept multiple lines by pre-commit, so I just simple add `r` to fix it, is there any good solution to prevent pre-commit error with this DOC str in multiple lines?
the error from vscode is 
```
Use `r"""`  if any backslashes in a docstring  
```
from Ruff(D301).

Reply:
I would avoid reporting an example of each value, while writing the type.
An example below.